code/game/objects/items/weapons/clown_items.dm

```diff
@@ -39,6 +39,13 @@
 	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
 	if(user.client && (target in user.client.screen))
 		to_chat(user, "<span class='notice'>You need to take that [target.name] off before cleaning it.</span>")
+	else if(target == user && user.a_intent == "grab")

```

Comment:
No need, already checks if they're a Drask. Only human type mobs can fulfill that requirement